snforge_std/src/trace.cairo

```diff
@@ -2,48 +2,55 @@ use core::result::ResultTrait;
 use core::starknet::testing::cheatcode;
 use core::starknet::ContractAddress;
 
+/// Tree-like structure which contains all of the calls and sub-calls along with the results
 #[derive(Drop, Serde, PartialEq)]
 struct CallTrace {
-    entry_point: CallEntryPoint,
-    nested_calls: Array<CallTrace>,
+    entry_point: CallEntryPoint, /// The topmost entry point of this trace
+    nested_calls: Array<CallTrace>, /// All the calls that happened in the context of `entry_point`
```

Comment:
```suggestion
    /// The inner calls that happened as a result of calling the `entry_point`
    nested_calls: Array<CallTrace>,
```

Reply:
I don't think there's strict causality there. There could be other calls that happened because of this, but not in the scope of this call, so i think the previous wording is more accurate (i could change "context" to "scope" perhaps?)

Reply:
Yea, scope is better imo